tarantool/const.py

```diff
@@ -41,6 +41,7 @@
 
 IPROTO_GREETING_SIZE = 128
 IPROTO_BODY_MAX_LEN = 2147483648
+IPROTO_CHUNK=0x80
```

Comment:
const is a bit messy, but I think it should go after `IPROTO_FEATURES = 0x55` and with spaces around ` = `

Reply:
Done